example-deployment/deploy.sh

```diff
@@ -38,7 +38,7 @@ cd $(dirname $0)
 
 kubectl apply -f k8s/keycloak.yaml
 echo 'waiting for keycloak to be ready...'
-kubectl wait --for=condition=available deployment/keycloak
+kubectl wait --for=condition=available deployment/keycloak --timeout=10m
```

Comment:
It is rare for keycloak to take so long to set up, longer timeout means longer wait before if decide keycloak is :boom: . As you mentioned, the a small aws instance takes slightly longer than 30s, so if it is still not up in 2-3min its pretty safe to say that keycloak is :boom:.